packages/create-onchain/src/utils.ts

```diff
@@ -1,11 +1,41 @@
 import fs from 'fs/promises';
 import path from 'path';
 
-async function copyFile(src: string, dest: string) {
+
+const renameFiles: Record<string, string | undefined> = {
+  _gitignore: '.gitignore',
+  '_env.local': '.env.local',
+};
+
+const excludeDirs = ['node_modules', '.next'];
+const excludeFiles = ['.DS_Store', 'Thumbs.db'];
+
+export async function copyDir(src: string, dest: string) {
+  await fs.mkdir(dest, { recursive: true });
+  const entries = await fs.readdir(src, { withFileTypes: true });
+
+  for (const entry of entries) {
+    const srcPath = path.join(src, entry.name);
+    const destPath = path.join(dest, renameFiles[entry.name] || entry.name);
+
+    if (entry.isDirectory()) {
+      if (!excludeDirs.includes(entry.name)) {
+        await copyDir(srcPath, destPath);
+      }
+    } else {
+      if (!excludeFiles.includes(entry.name)) {
+        await optimizedCopy(srcPath, destPath);
+      }
+    }
+  }
+}
+
+
+async function optimizedCopyFile(src: string, dest: string) {
```

Comment:
nit(non-blocking): This wrapper function isn't doing anything, so we could just use fs.copyFile directly. I see it was like this before, though, so no biggie